src/audio/audioProcessor.js

```diff
@@ -31,38 +31,64 @@ export default class AudioProcessor {
       this.audible.connect(this.splitter);
       this.splitter.connect(this.analyserL, 0);
       this.splitter.connect(this.analyserR, 1);
-    }
 
-    // Initialised once as typed arrays
-    // Used for webaudio API raw (time domain) samples. 0 -> 255
-    this.timeByteArray = new Uint8Array(this.fftSize);
-    this.timeByteArrayL = new Uint8Array(this.fftSize);
-    this.timeByteArrayR = new Uint8Array(this.fftSize);
+      // Initialised once as typed arrays
+      // Used for webaudio API raw (time domain) samples. 0 -> 255
+      this.timeByteArray = new Uint8Array(this.fftSize);
```

Comment:
I had pulled these out of the `if (context) {` condition, because we still need to initialize the arrays before they get set in `updateAudio`

Reply:
Makes sense! I assumed I made an error while merging things